packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.js

```diff
@@ -123,34 +123,36 @@ export default class QuanticCaseClassification extends LightningElement {
   _isSuggestionsVisible = true;
   /** @type {number} */
   _maxSuggestions = 3;
+  /** @type {boolean} */
+  _renderingError = false;
 
   connectedCallback() {
     registerComponentForInit(this, this.engineId);
+    if (!this.coveoFieldName) {
+      this._renderingError = true;
+      console.error(
+        'the coveoFieldName is required, please introduce its value'
+      );
```

Comment:
I would suggest

> coveoFieldName is required, please set its value

Reply:
FYI we already do [this](https://github.com/coveo/ui-kit/blob/d1a3f3eab52b54837c3f8cce3f1cbca147669c4c/packages/quantic/force-app/main/default/lwc/quanticResultLabel/quanticResultLabel.js#L67) in the result template components so perhaps it would be good to remain consistent.

Reply:
I agree I'll update this to have something consistent with other components.